.github/workflows/publish_release_images.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish Release OCI Images
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 3 * * 6'
```

Comment:
It would be nice to be more intelligent about when to publish release images but for now, I think weekly is sufficient.

Reply:
Is there a manually?

Reply:
Is there a manually?

Reply:
Manual option: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
Will add a manual trigger. Should I keep the cron job or ditch it?

Reply:
I think we can ditch it

Reply:
We could trigger a webhook, whenever a plugin for a specific pulpcore version released. (Future PR idea...)